### ISSUE TITLE
Add learning rate decay scheduler

### ISSUE THREAD

**Issue by ljbkusters** [ljbkusters/YOLOv3-1D#17]
LR decay is usefull to speed up early parts of training while having lower LR at later epochs. This would be a handy addition.

**Comment by ljbkusters** [comment]
Closed by pr #23
